<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RR19-MB34.local" failures="6" tests="17" name="restassuredtests" time="22.346" errors="0" timestamp="2020-05-18T09:10:19 EDT">
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkContentType" time="0.006"/>
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkCookieGenerated" time="0.002"/>
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkResponseBodyNotNull" time="0.016"/>
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkResponseTime" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkServerType" time="0.002"/>
  <testcase classname="com.employeeapi.testcases.TC001_Get_All_Employees" name="checkStatusLine" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC002_Get_Single_Employee_record" name="checkContentLength" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [134] but found [75]">
      <![CDATA[java.lang.AssertionError: expected [134] but found [75]
at com.employeeapi.testcases.TC002_Get_Single_Employee_record.checkContentLength(TC002_Get_Single_Employee_record.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkContentLength -->
  <testcase classname="com.employeeapi.testcases.TC002_Get_Single_Employee_record" name="checkEmpID" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testcases.TC002_Get_Single_Employee_record.checkEmpID(TC002_Get_Single_Employee_record.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkEmpID -->
  <testcase classname="com.employeeapi.testcases.TC002_Get_Single_Employee_record" name="checkResponseTime" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC002_Get_Single_Employee_record" name="checkStatusCode" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [200] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at com.employeeapi.testcases.TC002_Get_Single_Employee_record.checkStatusCode(TC002_Get_Single_Employee_record.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="com.employeeapi.testcases.TC002_Get_Single_Employee_record" name="checkemployeeName" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testcases.TC002_Get_Single_Employee_record.checkemployeeName(TC002_Get_Single_Employee_record.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkemployeeName -->
  <testcase classname="com.employeeapi.testcases.TC003_Post_New_Employee_Record" name="getResponseBody" time="0.003"/>
  <testcase classname="com.employeeapi.testcases.TC003_Post_New_Employee_Record" name="getStatusCode" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC003_Post_New_Employee_Record" name="getStatusLine" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC004_Put_Employee_Record" name="checkResponseBody" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testcases.TC004_Put_Employee_Record.checkResponseBody(TC004_Put_Employee_Record.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResponseBody -->
  <testcase classname="com.employeeapi.testcases.TC004_Put_Employee_Record" name="checkStatusCode" time="0.001"/>
  <testcase classname="com.employeeapi.testcases.TC005_Delete_Employee_Record" name="checkResponseBody" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testcases.TC005_Delete_Employee_Record.checkResponseBody(TC005_Delete_Employee_Record.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkResponseBody -->
</testsuite> <!-- restassuredtests -->
